katas/content/distinguishing_unitaries/i_z/solution.md

```diff
@@ -0,0 +1,20 @@
+> As a reminder, $$Z = \begin{bmatrix} 1 & 0 \\ 0 & -1 \end{bmatrix}$$
+
+We won't be able to distinguish $I$ from $Z$ by applying them to the basis states, since they both leave the $|0\rangle$ state unchanged and add a phase to the $|1\rangle$ state: 
+
+$$I|0\rangle = |0\rangle, I|1\rangle = |1\rangle$$
+$$Z|0\rangle = |0\rangle, Z|1\rangle = -|1\rangle$$
+
+However, if we try applying these gates to a superposition of basis states, we'll start seeing a difference between the resulting states:
+
+$$I \big(\frac{1}{\sqrt2}(|0\rangle + |1\rangle)\big) = \frac{1}{\sqrt2}(|0\rangle + |1\rangle)$$
+$$Z \big(\frac{1}{\sqrt2}(|0\rangle + |1\rangle)\big) = \frac{1}{\sqrt2}(|0\rangle \color{blue}{-} |1\rangle)$$
```

Comment:
I would drop \color tag here, since it either colors all the formula until the end of line or impacts the spacing around the minus sign, and neither option looks good
```suggestion
$$Z \big(\frac{1}{\sqrt2}(|0\rangle + |1\rangle)\big) = \frac{1}{\sqrt2}(|0\rangle - |1\rangle)$$
```

Reply:
good to know!